lessons/Early-Math/translating/langs/en-us/pages/translation-table2.adoc

```diff
@@ -0,0 +1,22 @@
+= Translation Table (Part 2)
+
+++++
+<style>
+table {grid-auto-rows: 1fr;}
+</style>
+++++
+
+
+Fill in any missing spaces on the table below.
+
+[.FillVerticalSpace, cols="^.^1a,^.^4a,^.^4a,^.^4a", stripes="none", options="header"]
+|===
+| 	 | Math 							| Circle of Evaluation							| Words
+| @n | @math{24 -6}						| @showsoln{(coe '(- 24 6))}						| @ifsoln{Find the difference between 24 and 6.}
+| @n | @math{24 - (6 \div 3)}			| @showsoln{(coe '(- 24 (/ 6 3)))}					| @ifsoln{Subtract the quotient of 6 and 3 from 24.}
+| @n | @math{(24 - 6) \div 3}			| @showsoln{(coe '(/ (- 24 6) 3))}					| @ifsoln{Find one-third of the difference of 24 and 6.}
+| @n | @math{\frac {1}{2} \times 100}	| @showsoln{(coe '(* 1/2 100))} 			| @ifsoln{Find half of 100.}
+| @n | @math{(\frac {1}{2} \times 100) + 10} |@showsoln{(coe '(+ (* 1/2 100) 10))}	| @ifsoln{Find the sum of 10 and half of 100.}
+| @n | @math{\frac {1}{2} \times (100 + 10)}	|@showsoln{(coe '(* 1/2 (+ 100 10)))} | @ifsoln{Find half the sum of 100 and 10.}
+|===
+
```

Comment:
I'm curious as to why you decided that page 1 would vary which representation was provided but page 2 would always provide words? (I don't necessarily object, just checking in to make sure it was intentional.)

Reply:
Jumping between representations exercises a different muscle than always going in the same direction (math, to Circles, to words - as on the 2nd worksheet); the first worksheet also ensures that students are exposed to example responses of all the different representations.

(I do suspect that I had a more concrete reason for designing the worksheets in this way - it was definitely intentional! - but I can't quite put my finger on it now.)